src/main/java/Calculator.java

```diff
@@ -0,0 +1,73 @@
+
+import java.util.Scanner;
+
+public class Calculator {
+    public double calRes =0;
+    private final String regExp = "^[0-9]+";
+private static final Scanner SCANNER = new Scanner(System.in);
+    public Double getcalRes() {
+        return this.calRes;
+    }
+    public Calculator() {
+        String input = getInput();
+        String[] inputArray = split(input);
+        double calRes=calculate(inputArray);
+    }
```

Comment:
이 생성자는 이렇게 고치면 좋을 것 같습니다 :) 생성자 로직이 반복되네요!
```suggestion
    public Calculator() {
        String input = getInput();
        this(input);
    }
```